Cargo.toml

```diff
@@ -52,4 +52,6 @@ members = [
    "websocket-chat",
    "websocket-chat-broker",
    "websocket-tcp-chat",
+   "docker_sample",
+   "async-graphql-demo"
```

Comment:
`docker_sample` here should be removed as duplicated and `async-graphql-demo` should be ordered alphabetically.